### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_CreateConntionDataSrc_Keyboard:Tool tip is not accessible through keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5785]
Bug Description:
[Check out Keros!](
https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
GitHubTags: #A11yMAS; #A11yTCS; #MAS1.1.1; #MAS2.1.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio 
App Version: 1.8.0-insider (user setup)
VS Code 1.33.1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS:OS : MacOS Mojave 10.14.5

**Additional Details:**
Tool Name: keyboard
MAS Reference:MAS1.1.1,MAS2.1.1

**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to ->Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button.
3.Hover till the "External Data Source Name. "
4.Observed the tool tip on the sensory character.

**Actual:**
Tool tip on the sensory character is not accessible through key board.

**Expected:**
Tool tip should be accessible through keyboard.

**User Impact:**
User with motor disabilities will find it difficult to navigate the page.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
**MAS1.1.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

**MAS2.1.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Attachment for Reference:
[5785_[Accessibility]A11y_AzureDataStudio_CreateDatabaseMasterKey_keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3241241/5785_.Accessibility.A11y_AzureDataStudio_CreateDatabaseMasterKey_keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by hannahqin** [comment]
This is the same underlying issue as #5676 and #6730 (see comments on #6730). 

Filed with vs code as https://github.com/microsoft/vscode/issues/83245 and going through the external bug documentation process.

**Comment by hannahqin** [comment]
Closing as a duplicate of #6730. We'll be tracking all progress on external bug documentation there.
